wireapp/wire-ios-mono
# chore: update PINCache to use privacy manifest [WPB-8882]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-8882" title="WPB-8882" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-8882</a>  Check PINCache update for privacy manifest
  </td></table>
  <br />
 









### Issue

Because [PINCache](https://github.com/pinterest/PINCache) doesn't provide any releases since 2020, we need to pin the last commit on `master` branch.

Pinning helps to avoid unreviewed updates in out project instead using the branch itself.

### Testing

- Use the app in places where user images are cached.
- Upload a build to Apple to see if the review works as expected.

---

### Checklist
